lib/bolt/node/ssh.rb

```diff
@@ -68,12 +78,22 @@ def execute(command, options = {})
           raise "could not execute command: #{command.inspect}" unless success
 
           channel.on_data do |_, data|
-            result_output.stdout << data
+            if use_sudo && data == sudo_prompt
```

Comment:
How many times will this retry if the password is wrong?

Reply:
as many times as it's prompted